### DISCUSSION TITLE
LD360 digital works

### DISCUSSION THREAD

**Discussion by Chete10** [Nortank12/deepcool-digital-linux#7] [General]
Good night
It works correctly on LD360 but I can't get it to start after the reboot having enabled the service to start automatically after the reboot, any solution?
Greetings.

**Comment by Nortank12** [comment]
Thanks for the feedback!

If you set up the service by the description using systemd, run the following command after a reboot, and send me the output:
```bash
sudo systemctl status deepcool-digital
```

**Comment by Chete10** [comment]
Good afternoon.
Thank you very much for the quick response, I am attaching the screenshot to see if you can help me.

![Captura desde 2024-08-23 15-17-32](https://github.com/user-attachments/assets/43b5f81c-defd-4f22-8fb0-928d0e6d88ab)

Greetings.

**Comment by Chete10** [comment]
Good afternoon.

I managed to solve it, I had to run systemctl daemon-reload, and then systemctl restart deepcool-digital.service.

Checked that after restarting the computer it works.

Thank you so much.

Greetings.

**Comment by Chete10** [comment]
Good afternoon.

I managed to solve it, I had to run systemctl daemon-reload, and then systemctl restart deepcool-digital.service.

Checked that after restarting the computer it works.

Thank you so much.

Greetings.
